### DISCUSSION TITLE
我觉得这个方法能解决一部分雷电adb问题

### DISCUSSION THREAD

**Discussion by Hammer2333** [LmeSzinc/AzurLaneAutoScript#2538] [General]
起因：夜神真的很卡，而其我想远程操控我的模拟器，所以pass了蓝叠
方法：1.雷电设置开启网络桥接模式
           2.然后本地adb先连接雷电模拟器，输入```adb tcpip 端口``` (需要shell权限)
           或者找一个安卓终端模拟器，获取root权限后输入```setprop service.adb.tcp.port 端口```
结果：开启网络adb调试，即局域网设备都能连接到这个模拟器，效果如图
![image](https://user-images.githubusercontent.com/80084984/235282714-92b152b6-daec-42c9-bf64-d2d637d9cf52.png)
最后：让Alas连接这个ip+端口就行了，目前很稳定，缺点要adb经过要过路由器再连回本地，可能要慢一些，而且其余未知问题我也没有遇到过（主要是雷电真的好流畅，就是想用雷电）
![image](https://user-images.githubusercontent.com/80084984/235282789-8257c566-a61b-44e1-83c9-f1116e63a08f.png)

**Comment by guoh064** [comment]
我自己挺需要tcpip模式的（因为有的时候我在wsl里面调试alas，随便接个adb设备就行，用wsa最快但是要跨子网）
期待你的pr

**Reply by LostMyMind** [reply]
> 我自己挺需要tcpip模式的（因为有的时候我在wsl里面调试alas，随便接个adb设备就行，用wsa最快但是要跨子网） 期待你的pr

发错地方了，抱歉

**Comment by LostMyMind** [comment]
关于雷电触控的问题，雷电客服给我回复了，说maatouch和adb可以保证没问题，似乎是专门找了maatouch的开发者来优化。本来也想优化minitouch，但是找不到minitouch的开发者，就尬在那了。

**Comment by bookbug666** [comment]
> 缺点要adb经过要过路由器再连回本地，可能要慢一些

@LostMyMind 有沒有試過創建一個回送網卡呢
https://blog.csdn.net/qq_33292195/article/details/121678965
然後因為微軟有改名，W8以後名字叫做[這個](https://learn.microsoft.com/zh-CN/troubleshoot/windows-server/deployment/microsoft-loopback-adapter-rename)

我在之前就將網卡設定為10.1.1.11，用腳本啟動雷電跟alas選MaaTouch也沒斷觸
提供一個不用經過實體路由器的辦法

~~沒有改變`遠端adb`選項對全網卡暴露adb裝置的風險，你填入你本地區域網路ip+port依然可以連接，就是讓你不用回路由而已~~
